src/main/java/be/syntra/devshop/DevshopFront/models/dtos/CartDto.java

```diff
@@ -15,13 +14,12 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
-@ToString
 @Component
```

Comment:
@Component?

Reply:
@Component is what replaced the bean in the configuration file.